modules/csharp/ROOT/examples/Program.cs

```diff
@@ -205,6 +205,7 @@ private static void TestImportTLSIdentity()
                 "CBL-Client-Cert", 
                 null); // The key label to get cert in certificate map. If null, the same default directory for a Couchbase Lite database is used for map.
             // end::import-tls-identity[]
+        }
```

Comment:
@ibsoln When you merge the prs, I think you forgot about this closing bracket.

Reply:
thanks